outline.md

```diff
@@ -0,0 +1,59 @@
+
+## Audience
+This presentation will be given at [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).
+It is intended for those that are evangelizing inner source in a specific company.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
+Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
+
+When building an inner source program you should have a vision of what services you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples that can aid and guide in doing so.
+
+The services of the inner source program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
+Examples of these services include:
+
+* Documentation on how to do inner source (and relation of that documentation to [InnerSourcePatterns](http://innersourcecommons.org/patterns)).
+* Software for automated, two-way discovery of inner source projects (maintainer to contributor/user and contributor/user to maintainer).
+* Software for a data-driven understanding of the impact of inner source projects.
+* Software/convention to facilitate individuals communicating with one another.
+* Curated training on the above software and documentation.
+
+The rollout of an inner source program is similar to the rollout of any large, grass-roots initiative.
+Documented patterns in this area (well-summarized in [the Fearless Change series](http://www.fearlesschangepatterns.com/)) apply and work.
+We'll review real examples of how these patterns played out in our journey.
+We'll also share a few patterns unique to inner source rollout including:
+
+* Run your inner source program as inner source projects.
+* Do (or say) nothing save it is documented.
+* Program growth/maturity progression starts with facilitating any synchronous, in-person, side-project talking collaboration then grows to facilitating asynchronous, main-project coding collaboration.
+
+After explaining these principles with clear words and real anecdotes, we'll summarize the current state of our inner source program, where we're hoping it will go, and how we aim to get there. We'll express our hope to learn from and work with everyone present via open dialogue and sharing in the inner source commons.
```

Comment:
There are principles missing from this list. I think that some of these are overlook as they are part of the teaching, defining and documenting the culture part.  However I think there are principles that are overlooked because they seem obvious to us.  

- Precise discovery mechanisms and patterns for shared knowledge and shared artifacts.
- Systematic and regular outreach for "drivers".
- Creating and using an economic argument for the value of community.
- Defining easy and reusable patterns (that are documented)
- Zen like action, never moving in a direction that creates waste or more than is needed.  Following the (Just Enough) principle.

Reply:
Is it fair to say that those are included (at an outline level) in this statement:

> Documented patterns in this area (well-summarized in [the Fearless Change series](http://www.fearlesschangepatterns.com/)) apply and work

And in the actual presentation we can include exposition of the ideas you've listed out specifically?

Reply:
Sure ok!  Yea I didn't get much out of the statement referencing the Fearless Change series.  I'm not sure what level of experience your audience will have with that book, and the while the patterns describe behaviors, I'm not sure that they all describe actionable steps to take.